requirements-dev.txt

```diff
@@ -52,3 +52,6 @@ fastparquet<2023.1.0
 # for release script
 PyGithub
 pygit2
+moto
```

Comment:
Should we add new deps to all env files?

Reply:
Added!